lib/checks/keyboard/has-no-more-than-one-banner.js

```diff
@@ -0,0 +1,29 @@
+const banners = axe.utils.querySelectorAll(virtualNode, 'header, [role=banner]');
+const sectioning = ['article','aside','main','nav','section'];
+var count = 0;
+
+function isBanner(node){
+  var parent = axe.commons.dom.getComposedParent(node);
+  while (parent){
+    if (sectioning.includes(parent.tagName.toLowerCase())){
```

Comment:
I don't quite understand what this is supposed to do. So if a `<header>` is contained in a `<main>`, it's ignored? Why is that?